processing/src/main/java/io/druid/segment/BitmapOffset.java

```diff
@@ -51,8 +52,9 @@ private IntIterator newIterator()
     if (!descending) {
       return bitmapIndex.iterator();
     }
-    // ImmutableRoaringReverseIntIterator is not cloneable.. looks like a bug
-    // update : it's fixed in 0.5.13
+    if (bitmapIndex instanceof WrappedImmutableRoaringBitmap) {
+      return ((WrappedImmutableRoaringBitmap) bitmapIndex).getBitmap().getReverseIntIterator();
+    }
     int i = bitmapIndex.size();
     int[] back = new int[bitmapIndex.size()];
```

Comment:
I didn't notice this earlier, but materializing the integer array might create a lot of heap pressure. Instead, why not copy the bitmap into a temporary roaring bitmap and use its reverse iterator?